docs/io/configuration/components/supernova.rst

```diff
@@ -4,9 +4,24 @@
 Supernova Configuration
 ***********************
 
-The supernova component of the configuration file contains some key information about the supernova being modeled:
+The supernova component of the configuration file contains some key information about the supernova being modeled, namely the time since the supernova and the luminosity the user wishes TARDIS should output:
```

Comment:
namely the time since the supernova explosion (which is used throughout the TARDIS calculations) and the luminosity....